default.json

```
@@ -0,0 +1,29 @@
+{
+  "labels": ["maintenance"],
+  "extends": [
+    "config:base",
+    ":disableDependencyDashboard"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": ["before 5am on monday"]
+  },
+  "rangeStrategy": "replace",
+  "semanticCommitType": "build",
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "semanticCommitType": "chore"
+    },
+    {
+      "depTypeList": ["dependencies", "peerDependencies"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "depTypeList": ["action"],
+      "semanticCommitType": "ci",
+      "semanticCommitScope": "action"
+    }
+  ],
+  "ignoreDeps": ["@pika/pack"]
+}
```

I would use this instead:

```suggestion
{
  "extends": ["github>octokit/.github"],
  "packageRules": [
    {
      "matchDepTypes": [
        "action"
      ],
      "pinDigests": true
    }
  ]
}
```
[disabling `helpers:pinGithubActionDigests`](https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigests)